rmw_connext_cpp/src/functions.cpp

```diff
@@ -1878,7 +1878,10 @@ rmw_get_topic_names_and_types(
   rmw_topic_names_and_types_t * topic_names_and_types)
 {
   return get_topic_names_and_types(rti_connext_identifier, node,
-           topic_names_and_types);
+           topic_names_and_types,
+           ros_topics_prefix,
+           ros_service_requester_prefix,
+           ros_service_response_prefix);
```

Comment:
I left it like this on purpose for not blowing up the diff. The style format here should be excluded in a different PR IMO.